README.md

```diff
@@ -66,7 +78,7 @@ The errors statuses can be:
 | HTTP Status | Reason                                                                                        |
 | ----------- | --------------------------------------------------------------------------------------------- |
 | 401         | Incorrect secret key                                                                          |
-| 400         | Payload too big (10Kb limit), Payload invalid, Payload in a wrong schema, instance_id missing |
+| 422         | Payload too big (10Kb limit), Payload invalid, Payload in a wrong schema, instance_id missing |
```

Comment:
We also need a 429.

Maybe we should point to the complete (and always up-to-date) list: https://docs.pusher.com/beams/reference/publish-api#error-responses

Reply:
Done in 7d2b898.